canister/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 [dependencies]
 bitcoin = "0.27.1"
 byteorder = "1.4.3"
-ic-btc-types = { path = "../types" }
+ic-btc-types = { path = "../../library/types" }
```

Comment:
I don't think this will work. We're referencing a path that is outside of this repo. If someone doesn't have the `library` repo cloned in that folder, the compilation will fail. For this to work, we'd need to import the library as a git repo. See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories for examples.